riptide-stream/src/main/java/org/zalando/riptide/stream/StreamSpliterator.java

```diff
@@ -86,4 +86,13 @@ public long estimateSize() {
     public int characteristics() {
         return ORDERED | IMMUTABLE;
     }
+
+    @Override
+    public void close() throws UncheckedIOException {
```

Comment:
`AutoCloseable` allows to throw any exception so there should be no need to wrap a checked exception here. But as Willi said, you could just use the `JsonParser#close` method as the `onClose` handler.